.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- Mark items with [x] where applicable -->
+#### Have the results of the proposed changes been tested?
+- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
+- [ ] I generally don't use the affected packages but briefly tested this PR
+- [ ] I did not test this PR
+
+<!-- If Travis cannot be used to validate the build result (for example if the
+build exceeds the timecap of 45min), make sure to put "[ci skip]" in the HEAD
+commit, which skips travis builds completely (saves CI resources). Uncomment the
+following section in this case. -->
```

Comment:
> If GitHub CI cannot be used to validate the build result (for example, if the build is likely to take several hours), make sure to include "[ci skip]' in either the PR title or this description. When skipping CI, uncomment and complete the following section. Note: for builds that are likely to complete in less than XXX hours, it is not acceptable to skip CI.

(I don't know the time limit, but XXX needs to be filled in.)

Reply:
The hard limit is 6 hours.

Reply:
thanks